lace/emulator/p1d_archive.py

```diff
@@ -147,7 +158,10 @@ def _load_data(self,drop_tau_rescalings,drop_temp_rescalings,
                 snap_p1d_data['Delta2_p'] = linP_params['Delta2_p']
                 snap_p1d_data['n_p'] = linP_params['n_p']
                 snap_p1d_data['alpha_p'] = linP_params['alpha_p']
-                snap_p1d_data['f_p'] = linP_params['f_p']
+                if 'f' in linP_params:
+                    snap_p1d_data['f_p'] = linP_params['f']
+                else:
+                    snap_p1d_data['f_p'] = linP_params['f_p']
```

Comment:
Same here. This looks like old code that we changed a while ago. I don't think that we should have archives with 'f' instead of 'f_p' anymore.